x-pack/plugins/siem/public/components/open_timeline/delete_timeline_modal/delete_timeline_modal.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { pure } from 'recompose';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import * as i18n from '../translations';
+
+const DeleteTimelineModalContainer = styled(EuiPanel)<{ width: number }>`
+  width: ${({ width }) => `${width}px`};
+  user-select: none;
+`;
+
+const CancelFlexItem = styled(EuiFlexItem)`
+  margin-right: 12px;
+`;
+
+interface Props {
+  title?: string | null;
+  onDelete: () => void;
+  toggleShowModal: () => void;
+}
+
+export const DELETE_TIMELINE_MODAL_WIDTH = 600; // px
+
+/**
+ * Renders a modal that confirms deletion of a timeline
+ */
+export const DeleteTimelineModal = pure<Props>(({ title, toggleShowModal, onDelete }) => (
+  <DeleteTimelineModalContainer paddingSize="l" width={DELETE_TIMELINE_MODAL_WIDTH}>
```

Comment:
Any reason you didn't use [EuiConfirmModal](https://elastic.github.io/eui/#/layout/modal) within this component?